src/react-apps/templates/template-app/src/types/global.ts

```diff
@@ -1,11 +1,11 @@
-import {IDashboardState} from '../reducers/dashboardReducer';
+import { ITemplateState } from '../reducers/templateReducer';
 
 declare global {
-  export interface IDashboardNameSpace<T1> {
+  export interface ITemplateNameSpace<T1> {
     dashboard: T1;
   }
 
-  export interface IServiceDevelopmentAppState
-    extends IDashboardNameSpace
-    <IDashboardState> { }
+  export interface ITemplateAppState
+    extends ITemplateNameSpace
+    <ITemplateState> { }
```

Comment:
Rename from "state" to "store"?
We do not want to confuse redux store with react component state.